dashboard/models.py

```diff
@@ -37,8 +37,10 @@ def __str__(self):
 
     # # Replace the year in the end date with start date (Hack to get around far out years)
     # # This should be replaced in the future via an API call so the terms have correct end years, or Canvas data adjusted
-    def get_correct_date_end(self):
-        if self.date_start is not None and (self.date_end.year - self.date_start.year) > 1:
+    def get_correct_date_end(self) -> models.DateTimeField:
```

Comment:
I think the type hint should be `-> Optional[datetime]` or `Union[datetime, None]`. It might be possible to use the model field like you are doing, but in reality, we're going to be returning either `None` or a `datetime` object.

Reply:
Thanks. Updated to Optional[datetime] now.